db/schema.rb

```diff
@@ -19,8 +19,19 @@
     t.string "title"
   end
 
+  create_table "reviews", force: :cascade do |t|
+    t.string "reviewer"
+    t.text "body"
+    t.bigint "book_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.integer "rating"
+    t.index ["book_id"], name: "index_reviews_on_book_id"
+  end
```

Comment:
I'd expect this to be associated with a User rather than have a `reviewer` column.